db/migrate/20151210225310_create_test_client_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTestClientRequests < ActiveRecord::Migration
```

Comment:
yes that is the main issue. 

I generally think it makes sense to write test for code and not to write code for tests. 

Tests can uncover places where code is not written well and nudge you toward a better implementation (eg: if a test requires a ton of setup, that is a good indicator that your object has too many dependencies) but going as far as creating a migration for a test seems like writing code for tests rather than writing better code so it is more easily testable.

On the other hand, I agree that it would be weird to have the test db schema not match the regular schema. Not sure what other options are. This may be another case where having the ability to instantiate a generic client data object would be helpful.

Thoughts?